.evergreen/serverless/create-instance.sh

```diff
@@ -46,6 +46,17 @@ if [ -z "$SERVERLESS_INSTANCE_NAME" ]; then
     SERVERLESS_INSTANCE_NAME="$RANDOM-DRIVERTEST"
 fi
 
+# Ensure that a Python binary is available for JSON decoding
+# shellcheck source=.evergreen/find-python3.sh
+. ../find-python3.sh || return
+
+PYTHON_BINARY="$(find_python3)"
+
+if [ -z "$PYTHON_BINARY" ]; then
+    echo "Failed to find Python3 binary"
+    exit 1
+fi
```

Comment:
Previously, the script might have spawned a serverless instance via `curl` and _then_ failed due to a bad Python binary path. Moving this up just ensures we abort the script earlier before issuing any API requests.